zotero2readwise/readwise.py

```
@@ -175,7 +298,13 @@ def post_zotero_annotations_to_readwise(self, zotero_annotations: list[ZoteroIte
         )
         print(finished_msg)
 
-    def save_failed_items_to_json(self, json_filepath_failed_items: str = None):
+    def save_failed_items_to_json(self, json_filepath_failed_items: str = None) -> None:
```

Ruff (0.14.5)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)